packages/shared/styles/components/SfAddToCart.scss

```
@@ -0,0 +1,30 @@
+@import '../variables';
+
+.sf-add-to-cart {
+  display: flex;
+
+  &__button {
+    &:disabled {
+      background-color: $c-accent-secondary;
+      cursor: default;
+    }
+  }
+
+  &__select-quantity {
+    margin-left: 0.5rem;
+
+    input {
+      padding-left: 0.5rem;
+      padding-right: 0.5rem;
+      width: auto;
+      height: 100%;
+      max-width: 90px;
+      border: 1px solid $c-light-primary;
+      transition: all 0.3s ease-in-out;
```

```suggestion
      transition: border-color 0.3s ease-in-out;
```
is more efficient when you used only property with you change :)